.travis.yml

```diff
@@ -37,11 +37,13 @@ env:
 cache:
   directories:
     - $HOME/.composer/cache
+    - build/cache
```

Comment:
seems like it's not cached: https://travis-ci.org/infection/infection/jobs/354907585#L917

Reply:
Hard to tell why it does not in this specific build, but [see here](https://travis-ci.org/sanmai/pipeline/jobs/354201230#L771) and [here](https://travis-ci.org/kalimatas/php-liquid/jobs/339277988#L728) where the very same scheme works as expected.

Reply:
I think it needs to be an absolute path but the cache won't kick in with the PR, it needs to be merged to see it in effect

Reply:
Well, the opposite. [Docs say it must be relative](https://docs.travis-ci.com/user/caching#Arbitrary-directories) in our case.